00 Intro/01 Import/README.md

```diff
@@ -1,149 +1,174 @@
+  ![running webpack 2](../../99%20Readme%20Resources/00%20Intro/01%20Import/result.png)
 
-```diff
-- export function getAvg(scores) {
-+ export default function getAvg(scores) {
- return getTotalScore(scores) / scores.length;
-}
+## Appendix - Module alternative usage
 
-function getTotalScore(scores) {
-  return scores.reduce((score, count) => {
-    return score + count;
-  });
-}
+We have covered a single named export usage in our previous example, but there are some other ways to use modules:
 
-```
+### Default export
+  
+  One popular way is using **`export default`** as the export keyword. This will indicate that, by default, there will be just a **single export per module**. Then, we can directly use an import *alias* (by omitting the curly braces) and this will point out to our default exported member (*averarge* function in our example).
 
-Import and usage (students.js)
+  - Default export usage in `averageService.js`:
 
-```diff
-- import {getAvg} from "./averageService";
-+ import getAvg from "./averageService";
+    ```diff
+    - export function getAvg(scores) {
+    + export default function getAvg(scores) {
+    return getTotalScore(scores) / scores.length;
+    }
 
-const scores = [90, 75, 60, 99, 94, 30];
-const averageScore = getAvg(scores);
+    function getTotalScore(scores) {
+      return scores.reduce((score, count) => {
+        return score + count;
+      });
+    }
 
-const messageToDisplay = `average score ${averageScore}`;
+    ```
 
-document.write(messageToDisplay);
-```
+  - Default import usage in `students.js`:
 
+    ```diff
+    - import {getAvg} from "./averageService";
+    + import getAvg from "./averageService";
 
-- Another way to use import is to use `*` to indicate we want to import everything
-from that module, let's add for the sake of this sample two functions sum and average,
-then on the main students file we can import everything provides an alias and use
-the function we need.
+    const scores = [90, 75, 60, 99, 94, 30];
+    const averageScore = getAvg(scores);
 
-Several Exports (AverageService)
+    const messageToDisplay = `average score ${averageScore}`;
 
-```diff
-- export default function getAvg(scores) {
-+ export function getAvg(scores) {
- return getTotalScore(scores) / scores.length;
-}
+    document.write(messageToDisplay);
+    ```
 
-- function getTotalScore(scores) {
-+ export function getTotalScore(scores) {
-  return scores.reduce((score, count) => {
-    return score + count;
-  });
-}
+### Multiple named exports
 
+Let's consider two functions, *getAvg* and *getTotalScore*, for the sake of this example. We can export both using named exports, just by adding the **export** keyword on each function. 
 
-```
+  - Multiple exports usage in `averageService.js`:
 
-Import * and usage sum + average
+    ```diff
+    - export default function getAvg(scores) {
+    + export function getAvg(scores) {
+    return getTotalScore(scores) / scores.length;
+    }
 
-```diff
```

Comment:
I don't get this point either Dani :(. The code block you refer to is already preceded by the caption:

Multiple exports usage in averageService.js:
Do you mean to add the path of the file (e.g.: ./averageService.js) on top of each code block?